src/miner.cpp

```diff
@@ -134,7 +135,7 @@ CBlockTemplate* BlockAssembler::CreateNewBlock(const CScript& scriptPubKeyIn)
                        : pblock->GetBlockTime();
 
     addPriorityTxs();
-    addScoreTxs();
+    addPackageTxs();
```

Comment:
I raised that question at the beginning of this pull, and no discussion has happened; I'd like to defer that to it's own PR to give anyone who is clinging to the old behavior a chance to argue for keeping the code?